docs/methods/NamedAlleleMatcher-101.md

```diff
@@ -28,12 +28,12 @@ Take a look at this sample gene definition table:
 
 |     | rs1 | rs2 | rs3 | rs4 | rs5 | score |
 | --- | --- | --- | --- | --- | --- | ----- |
-| *1  | C   | C   | T   | G   | A   | 5     |
-| *2  | T   | T   |     | A   |     | 3     |
+| \*1 | C   | C   | T   | G   | A   | 5     |
+| \*2 | T   | T   |     | A   |     | 3     |
 
-Since the gene definition table contains 5 positions, the reference allele, _*1_, gets a score of 5 while _*2_ only has 3 positions defined and gets a score of 3. <sup>[3](#notes)</sup>
+Since the gene definition table contains 5 positions, the reference allele, `*1`, gets a score of 5 while `*2` only has 3 positions defined and gets a score of 3. <sup>[3](#notes)</sup>
```

Comment:
No monospace or double quotes, just plain text on this line. The previous table escaping is good.